homeassistant/util/unit_system.py

```diff
@@ -120,6 +128,16 @@ def length(self, length: float | None, from_unit: str) -> float:
             length, from_unit, self.length_unit
         )
 
+    def accumulated_precipitation(self, precip: float | None, from_unit: str) -> float:
+        """Convert the given length to this unit system."""
+        if not isinstance(precip, Number):
+            raise TypeError(f"{precip!s} is not a numeric value.")
```

Comment:
Let's cover this in a test since this is really core functionality

Reply:
My bad. I made a typo in the tests, but it is covered now!